src/substrait/textplan/SymbolTable.cpp

```diff
@@ -12,7 +12,7 @@
 #include "substrait/textplan/Location.h"
 #include "substrait/textplan/StructuredSymbolData.h"
 
-namespace io::substrait::textplan {
```

Comment:
Would changing this to three separate namespace definitions did the problem?

Reply:
It's hard to find documentation about this (it's sort of a bug-in-limbo...). I'm not at my workstation at the moment so i can't give an answer now, but my hunch would be no.